ogc-sys/wrapper.h

```diff
@@ -4,4 +4,6 @@
 #include <stdlib.h>
 #include <network.h>
 #include <debug.h>
-#include <ogcsys.h>
\ No newline at end of file
+#include <ogcsys.h>
+#include <math.h>
```

Comment:
We do not need the `math.h` header for wpad. If we wanted math related function. I think we should use [libm](https://crates.io/crates/libm) instead.

Reply:
I believe `math.h` is included because otherwise the application just crashes on runtime because of missing functions, if I remember correctly.

Reply:
The crashing would be because you aren't linking libm. I dont recommmend we reexport ogc's math stuff.